### PR TITLE
fixes for grammar, punctuation and missing images

### PR DESCRIPTION
**Pull request by netrik182**
Hi @serhack!

In this PR you will find the typos and missing information I found while translating the book into Brazilian Portuguese.

I'm also leaving some comments at lines where I think something might be wrong or confusing. Please let me know if you don't understand what I tried to say.

### CODE REVIEW

File: chapters/4.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ When Monero was launched, the coin emission rate was more than 30 XMR every 2 mi
...
+The annual supply increase from Monero's 0.6 XMR tail emission is less than 1% per year. While developers created tail emission to incentivize miners long-term, a neat by-product is its easy comparison to asset backed financial systems. The tail emission rate of supply increase is comparable to the annual gold supply increase globally, which in modern civilization has historically been the purest and most stable form of money. Since the miners will always be able to collect fees, the guaranteed coinbase payments provide greater financial stability for the miners in the long run. This social contract ensures that the miners remain incentivized to use their equipment to secure the Monero network.
```
Comment (netrik182) [inline chapters/4.md]: Here I think saying "annual" and "per year" in the same sentence looks redundant.
My suggestion would be to rephrase it:

`The annual supply increase from Monero's 0.6 XMR tail emission is less than 1%.`

or

`The supply increase from Monero's 0.6 XMR tail emission is less than 1% per year.`

File: chapters/2.md [hunk 2]
```diff
@@ -204,7 +208,7 @@ To avoid loss due to theft, never share your seed or keys with anybody else. Any
 
 When sending any large transaction to a new person or exchange, you should always initially test the address/service by sending a smaller amount first. When sending any large transaction to a new person or address, you should always send a tiny pilot transaction first and wait for the recipient to confirm that it arrived. This is an important habit to catch mistakes in advance, since there is no “undo”  button for cryptocurrencies.
```
Comment (netrik182) [inline chapters/2.md:209]: Here the first and second sentences look very similar. I understand their differences though.
Maybe rephrase one of them?

**Review by netrik182 — COMMENTED** [review]
Chapter 2 Line 226: "When the minor CryptoNote tweak was implemented at block 1546000", I don't know if it should be "CryptoNight" or not. This nomenclature confuses me.

Chapter 5 Line 419: "(c1 , s1 , s2 , s2 , J)", here the term `s2` appears twice, not sure if it's right or wrong.

File: chapters/4.md (hunk 1 above)
Comment (serhack) [reply chapters/4.md]: ```The supply increase from Monero's 0.6 XMR tail emission is less than 1% per year.``` 
It seems a good choice for me.

**Comment by serhack** [comment]
Your repository should be based on `second-edition` branch.

**Comment by netrik182** [comment]
> Your repository should be based on `second-edition` branch.

I'll have to look into this next weekend. Do you think some of this comments I left are already resolved on `second-edition`? I didn't dig into it yet.

**Comment by serhack** [comment]
> Do you think some of this comments I left are already resolved on second-edition?

Maybe only one. I still appreciate this pull request!

**Comment by lubomr** [comment]
@netrik182  Since the PR is not merge yet, could you please add a space in ch1. "tostudy" should be fixed.
image
Thans in advance!

**Comment by lubomr** [comment]
There are duplicated words in chapter 4, section 4.2.3 “Local nodes” versus “remote nodes”. 
image

**Review by lubomr — CHANGES_REQUESTED** [review]
Please remove the repeated "mitigate these" in 4.2.3 “Local nodes” versus “remote nodes”.

**Review by lubomr — APPROVED** [review]
